config/Makefile.msvc64

```diff
@@ -94,7 +94,10 @@ BYTECC=cl -nologo -D_CRT_SECURE_NO_DEPRECATE
 BYTECCCOMPOPTS=-O2 -Gy- -MD
 
 ### Additional compile-time options for $(BYTECC).  (For debug version.)
-BYTECCDBGCOMPOPTS=-DDEBUG -Zi -W3 -Wp64
```

Comment:
`-Wp64` was deprecated a loooong time ago - `/W4` replaces it now. `-W3` causes a vast number of warnings to be displayed.